Lib/multiprocessing/resource_tracker.py

```diff
@@ -57,6 +57,7 @@ def __init__(self):
         self._lock = threading.Lock()
         self._fd = None
         self._pid = None
+        self._exitcode = None
```

Comment:
Should `self._exitcode` also be reset when the process is spawned in `ensure_running` below?

Reply:
Good catch, I'll update there too